Cargo.toml

```diff
@@ -15,11 +15,12 @@ required-features = ["cli"]
 
 [dependencies]
 env_logger = "0.6"
-lazy_static = "1.3"
+lazy_static = "1.4"
 log = "0.4"
-rbx_dom_weak = "1.8"
+rbx_binary = { git = "https://github.com/rojo-rbx/rbx-dom/", rev = "54531ee87511312fedfd3e3767d6b9f8d14a8a80" }
+rbx_dom_weak = { git = "https://github.com/rojo-rbx/rbx-dom/", rev = "54531ee87511312fedfd3e3767d6b9f8d14a8a80" }
 rbx_reflection = "3" # TODO: This won't always be latest, and that's an issue
-rbx_xml = "0.10"
+rbx_xml = { git = "https://github.com/rojo-rbx/rbx-dom/", rev = "54531ee87511312fedfd3e3767d6b9f8d14a8a80" }
```

Comment:
I'd recommend using `branch = "master"` here instead of pinning a specific revision. You can let the lockfile handle keeping the revision for you, which will make later updates easier.